tzdata/Readme.txt

```diff
@@ -1,2 +1,63 @@
-The data files in this directory were obtained from the public IANA time zone database,
-http://www.iana.org/time-zones, version 2016d.
+README for the tz distribution
```

Comment:
I don't replace the README on purpose, instead using it as a reference for the files.